File: Java_Basic_Jan_2023_SolutionsOfProblems/P02LAB_ConditionalStatements/P04_PasswordGuess.java
package P02LAB_ConditionalStatements;

import java.util.Scanner;

public class P04_PasswordGuess {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String password = "s3cr3t!P@ssw0rd";
        String attempt = scanner.nextLine();
        if (attempt.equals(password))
            System.out.println("Welcome");
        else
            System.out.println("Wrong password!");
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.LAB_PB-JS-Conditional-Statements/P04_PasswordGuess.js
function passwordGuess(input) {
    let passW = input[0];
    if (passW === "s3cr3t!P@ssw0rd") {
        console.log("Welcome");
    } else {
        console.log("Wrong password!");
    }
}

passwordGuess(["s3cr3t!P@ssw0rd"]);